wordpress-mobile/WordPress-iOS
# [Stats Refresh] DWMY Async loading and Ghost views

Refs. #12147 

This PR continues the Load Blocks Independently started [here](https://github.com/wordpress-mobile/WordPress-iOS/pull/12764), implementing a new operation, new operations in the queue and ghost views. It also refactored the method used to handling the blocks in Insights.

To keep the PR easy to review and because of this refactor, I implemented the operations and the ghost views for these blocks only:
• _summary_
• _topPostsAndPages_
• _topReferrers_
• _topPublished_
• _topClicks_

_topPublished_ uses a different API. For this reason there's a specific operation for that case: `StatsPublishedPostsAsyncOperation`

### Refactor
The method implemented by the Insights and Period view models is now a protocol called `AsyncBlocksLoadable`. It uses a Generic Value as return type, which means it can be anything (in our case a single or a collection of ImmutableRow).

## To test:
- Open Stats -> DWMY from a Site you never opened before (or from a fresh install)
- Yous shouldn't see the NRV
- You should be able to load the chart and Top Posts card
- Disable the feature flag and make sure everything works as before

Update release notes:
